### PR TITLE
dts: Add new panel Xiaomi Mi 4i (xiaomi-ferrari)

### PR DESCRIPTION
**Pull request by KetanM94**
PANEL - qcom,mdss_dsi_jdi_nt35595_1080p_video

### CODE REVIEW

File: dts/msm8916/msm8939-mtp.dts [hunk 1]
```diff
@@ -16,8 +16,22 @@
 	};
 
 	xiaomi-ferrari {
+		/* Included becuase log said qcom,msm-id entry not found and this made it recognize the device tree correctly */
+		qcom,msm-id = <239 0>;
+		qcom,board-id = <8 0>;
```
Comment (Minecrell) [inline dts/msm8916/msm8939-mtp.dts:21]: This shouldn't be necessary... Does the display show an error? Don't worry about the log.

File: dts/msm8916/msm8939-mtp.dts [hunk 2]
```diff
@@ -16,8 +16,22 @@
...
+		
 		model = "Xiaomi Mi 4i";
 		compatible = "xiaomi,ferrari", "qcom,msm8939", "lk2nd,device";
-		lk2nd,match-cmdline = "* mdss_mdp.panel=1:dsi:0:qcom,mdss_dsi_sharp_rsp61322_1080p_video:1:none";
```
Comment (Minecrell) [inline dts/msm8916/msm8939-mtp.dts:21]: Need to replace this with `lk2nd,match-panel;`

File: dts/msm8916/msm8939-mtp.dts [hunk 3]
```diff
@@ -16,8 +16,22 @@
...
+
+		panel {
+
```
Comment (Minecrell) [inline dts/msm8916/msm8939-mtp.dts:27]:
```suggestion
```

File: dts/msm8916/msm8939-mtp.dts [hunk 4]
```diff
@@ -16,8 +16,22 @@
...
+			qcom,mdss_dsi_sharp_rsp61322_1080p_video {
+				compatible = "sharp,rsp61322";
```
Comment (Minecrell) [inline dts/msm8916/msm8939-mtp.dts:29]: Please use more unique names like

```suggestion
				compatible = "xiaomi,sharp-rsp61322";
```

File: dts/msm8916/msm8939-mtp.dts [hunk 5]
```diff
@@ -16,8 +16,22 @@
...
+			};
+
+			qcom,mdss_dsi_jdi_nt35595_1080p_video {
+				compatible = "jdi,nt35595";
```
Comment (Minecrell) [inline dts/msm8916/msm8939-mtp.dts:33]:
```suggestion
				compatible = "xiaomi,jdi-nt35595";
```

**Review by Minecrell — CHANGES_REQUESTED** [review]

File: dts/msm8916/msm8939-mtp.dts (hunk 1 above)
Comment (KetanM94) [reply dts/msm8916/msm8939-mtp.dts:21]: Ok. The display works fine. It was just log.

File: dts/msm8916/msm8939-mtp.dts (hunk 2 above)
Comment (KetanM94) [reply dts/msm8916/msm8939-mtp.dts:21]: Ok

File: dts/msm8916/msm8939-mtp.dts (hunk 3 above)
Comment (Minecrell) [reply dts/msm8916/msm8939-mtp.dts:24]: Can you drop this empty line?

Comment (Minecrell) [reply dts/msm8916/msm8939-mtp.dts:24]: It's still there :D

Comment (KetanM94) [reply dts/msm8916/msm8939-mtp.dts:24]: Sure!

**Pull request merged**